src/supertux/command_line_arguments.cpp

```diff
@@ -355,6 +356,17 @@ CommandLineArguments::parse_args(int argc, char** argv)
         repository_url = argv[++i];
       }
     }
+    else if (arg == "--open-in-editor")
+    {
+      if (i + 1 >= argc)
+      {
+        throw std::runtime_error("Need to specify a level for --open-in-editor");
+      }
+      else
+      {
+        editor_level = argv[++i];
```

Comment:
If we make `--edit-level` consistent with the current naming style, this could be `edit_level` (similar to `start_demo`, `play_demo`, and `start_level`).